### ISSUE TITLE
Bootstrap minimal website template

### ISSUE THREAD

**Issue by girikuncoro** [cloudnative-id/website#1]
This should be done using Gatsby or Hugo, in order to ease deployment from Git repo.

Reference:
https://cloudnativenordics.com/

**Comment by kamalhm** [comment]
Is the Gatsby or Hugo a must? considering the reference use Nuxt, if deployment is a concern, AFAIK Netlify/Zeit can be used to ease the deployment directly from git repo

**Comment by irvifa** [comment]
Yes, I also consider using netlify for this. As long as it’s supported by netlify it’s ok. However the example given here is what I guess more familiar to the people we have there.

**Comment by girikuncoro** [comment]
I think one of the goals is to be able render static pages from markdown files as well (like https://github.com/kubernetes/website), I'm not an expert on this, feel free to suggest better toolings/frameworrk

**Comment by joshuabezaleel** [comment]
If I am not mistaken, static site like Hugo can also be used along with Netlify directly from the GitHub repo. 
https://gohugo.io/hosting-and-deployment/hosting-on-netlify/
